Source/JavaScriptCore/runtime/AtomicsObject.cpp

```diff
@@ -48,6 +50,7 @@ STATIC_ASSERT_IS_TRIVIALLY_DESTRUCTIBLE(AtomicsObject);
     macro(store, Store, 3)                                              \
     macro(sub, Sub, 3)                                                  \
     macro(wait, Wait, 4)                                                \
+    macro(waitAsync, WaitAsync, 4)                                      \
```

Comment:
We should expose it only when `Options::useAtomicWaitAsync` is true. And we should add that option flag to `OptionsList.h`. (See `useArrayGroupMethod` for example).

Reply:
Still working on this one.

Reply:
Fixed